backtracepython/__init__.py

```diff
@@ -9,44 +8,32 @@
 
 import simplejson as json
 
+from backtracepython.attributes.attribute_manager import AttributeManager
+
+from .version import version, version_string
```

Comment:
`.version`? is this correct?

Reply:
yeah thats correct. This is how I can refer to the file at the same directory. Do you have a better option?

Reply:
if it works, it's okay. But let's make this consistent - can you use it in other cases?

Reply:
if I'm in the same module - yes. If I need to import a file from another module, I can't